lib/skeleton/license.rb

```diff
@@ -1,16 +1,5 @@
-require 'skeleton/model'
-
 module Skeleton
-  class License < Model
+  class License
```

Comment:
For some reason I enjoy boiler plate code. I could just have `class License < Struct.new(:url, :name)` to make sure it still is a class.